api/index.js

```diff
@@ -189,6 +189,7 @@ module.exports = async (req, res) => {
 
   const stats = await fetchStats(username);
 
+  res.setHeader('Cache-Control', 's-maxage=3600');
```

Comment:
I think 5min cache would be good enough since we need the frequent updates too

res.setHeader("Cache-Control", "public, max-age=300");

Reply:
Yep, [similar](https://github.com/fransallen/github-readme-stats/blob/master/api/index.js#L21)

Reply:
5 minutes would be good I guess, I made it one hour just to test it.

Reply:
Ah! magic number

Reply:
yup, good catch, I can move it into a constant or something.

```js
const FIVE_MINUTES = 300;
res.setHeader('Cache-Control', `public, max-age=${FIVE_MINUTES}`);
```

Reply:
1. Create a constants file under src, add all the constants there -> export -> tada

Reply:
yup, I commented that to Anurag, but he said he wanted to add that to env so it can easily be changed.

Reply:
Awesome